library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -526,20 +527,22 @@ impl From<c_int> for ExitStatus {
 impl fmt::Display for ExitStatus {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         if let Some(code) = self.code() {
-            write!(f, "exit code: {}", code)
+            write!(f, "exit status: {}", code)?;
```

Comment:
I don't think this PR should change this particular bit of the output. We currently refer to this as `code` and provide a `.code()` function to retrieve it, so using the same consistent terminology seems preferable.

Reply:
Since this is controversial I will split it out into a different MR.  (The "consistent terminology" from the stdlib names is simply wrong for Unix, as I have recently documented properly.)

Reply:
I'm not trying to make a case for or against that terminology, just that I don't think that change should be in this PR.

Regarding the change itself, my only concern is not introducing a source of confusion by being inconsistent between documentation and existing function names.

Reply:
I can see why you have this concern.  I agree that the conversation would be better in a different MR, so yes, I will do that.  Thanks.  (I'll "resolve" this review comment with that other MR when it exists.)